bench/wtperf/wtperf.c

```diff
@@ -1410,19 +1411,12 @@ flush_tier_worker(void *arg)
         if (wtperf->error)
```

Comment:
I think we should remove the comment before this line ("If workers are done...").  It doesn't really apply to this line of code, and the idea that we want to do a final flush is already stated in the previous comment.

Reply:
The comments should be fixed to reflect that we do a checkpoint with flush_tier configuration. They imply the old API.